invisible_cities/reco/deconv_functions_test.py

```diff
@@ -2,6 +2,7 @@
 import numpy  as np
 import pandas as pd
 
+from pytest                  import raises
```

Comment:
```suggestion
from pytest                  import mark
from pytest                  import raises
```
In order to parametrize the test below